specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/expressRouteCircuit.json

```diff
@@ -1779,9 +1979,9 @@
           "type": "string",
           "description": "The GatewayManager Etag."
         },
-        "allowGlobalReach": {
+        "globalReachEnabled": {
```

Comment:
That looks like a breaking change... unfortunately this monthly branch allows additive changes only (because of NRP release delays we're still merging to already published api-version ```2018-12-01```. you can revert breaking change or wait for the next monthly release (hope it'll be new api version)

Reply:
This field was added in [A] but then we abandoned it in favor of globalReachEnabled field being added in the current PR. The abandoned powershell PR [B]. The field was never consumed and is a dead field.
[A] https://github.com/Azure/azure-rest-api-specs/pull/3542
[B] https://github.com/Azure/azure-powershell/pull/6845

Reply:
I've reverted the remove for allowglobalreach.